docs/src/main/paradox/docs/releases/v1.7-release-notes.md

```diff
@@ -0,0 +1,75 @@
+# v1.7 Release Notes
+
+> This is an incremental software release that adds several improvements across the board. The API is backwards
+> compatible to v1.6.
+
+For the detailed list of updates in this release, see the
+//TODO Update end date 
+@link:[list of addressed issues](https://github.com/BlueBrain/nexus/issues?q=is%3Aissue+is%3Aclosed+closed%3A2021-10-14..2022-02-01+){ open=new }
+since v1.6.
+
+If you are currently using Nexus Delta from 1.6.x, please
+visit the @ref:[migration instructions](v1.6-to-v1.7-migration.md) for the necessary steps to successfully upgrade.
+
+## Nexus Delta
+
+The main goal of this release is to provide support for an enhanced search experience in Fusion as well as to introduce
+a new extensible model to define Elasticsearch views based on pipes.
+
+Secondary goals include:
+
+//TODO update the links to 1.6.x
+
+- @link:[Deletion of the Nexus cli](https://bluebrainnexus.io/v1.5.x/docs/utilities/index.html#nexus-cli)
+- @link:[Deletion of the docker based client](https://bluebrainnexus.io/v1.5.x/docs/utilities/projections.html)
+- Allow to define an external file for Delta configuration
+- Allow deleting tags on resources
+- Allow tagging deprecated storages and views
+- Refactor `graph-analytics` plugin to make indexing faster
+
+A series of bugs have also been fixed with this new release.
+
+### Deprecations
+
+In an upcoming version, the support of Cassandra as a primary store for Nexus Delta will be removed.
+
+Nexus Delta will only focus on PostgreSQL to achieve this role as it better fits its needs.
```

Comment:
Maybe the following reads a bit better:

> In the upcoming version, the support of Cassandra as a primary store for Nexus Delta will be removed in favour of PostgreSQL to focus development efforts on features rather than supporting multiple databases. For very large deployments there are commercial options that are wire compatible to PostgreSQL.